src/views/clusteroverview/MonitoringTab/migrations-card/components/MigrationsTable/utils/filters.ts

```diff
@@ -0,0 +1,89 @@
+import { TFunction } from 'react-i18next';
+
+import { V1VirtualMachineInstance } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { RowFilter } from '@openshift-console/dynamic-plugin-sdk';
+
+import { vmimStatuses } from './statuses';
+
```

Comment:
please check for all those functions that receive t as argument if you can simply import the default T , not the one from the hook, directly into this file

Reply:
we can add a comment to utils/i18n.ts and remove this parameter, but I don't like that approach